src/components/views/rooms/EntityTile.js

```diff
@@ -29,6 +29,23 @@ var PRESENCE_CLASS = {
     "unavailable": "mx_EntityTile_unavailable"
 };
 
+
+function presence_class_for_member(presence_state, last_active_ago) {
```

Comment:
hm, doesn't this completely contradict our code style?

https://github.com/matrix-org/matrix-react-sdk/blame/develop/code_style.md#L35